public/app/features/dashboard-scene/serialization/transformToV1TypesUtils.ts

```diff
@@ -113,44 +113,52 @@ export function transformMappingsToV1(fieldConfig: FieldConfigSource): FieldConf
     }
   };
 
+  const transformedDefaults: any = {
```

Comment:
this is to fix the issue where when setting `defaults` to an empty object, we get:
```
defaults: {
    mappings: undefined,
    thresholds: undefined
}
```